lib/mayaUsd/fileio/utils/meshWriteUtils.cpp

```diff
@@ -477,4 +478,76 @@ UsdMayaMeshUtil::assignSubDivTagsToUSDPrim(MFnMesh& meshFn,
     }
 }
 
+void
+UsdMayaMeshUtil::writeVertexData(const MFnMesh& meshFn,
+                                 UsdGeomMesh& primSchema,
+                                 const UsdTimeCode& usdTime,
+                                 UsdUtilsSparseValueWriter& valueWriter)
+{
+    MStatus status{MS::kSuccess};
+
+    const uint32_t numVertices = meshFn.numVertices();
+    VtArray<GfVec3f> points(numVertices);
```

Comment:
I know the original code used the templated VtArray, but it's slightly preferred to use the named types for these:
`VtVec3fArray`
`VtIntArray`
etc.

Reply:
see 4ca16c08570949b8c0d73025d389139b03edbf30